README.md

```diff
@@ -31,8 +31,9 @@ This library will look for an environment variable,
 `IOPIPE_TOKEN` and will use this if one is not
 explicitly passed to the configuration object.
 
-This is an easy way to separate configuration from
-code.
+**N.B.** If you have `IOPIPE_TOKEN` set, _and_ pass a configuration object
```

Comment:
Isn't that what the above sentence is trying to say too? I'd prefer one concise sentence. i.e. "The lib will use the `token` configuration key and fall back to the environment variable `IOPIPE_TOKEN`"